exporters/trace/ocagent/src/test/java/io/opencensus/exporter/trace/ocagent/FakeOcAgentTraceServiceGrpcImpl.java

```diff
@@ -154,12 +164,10 @@ private synchronized void sendUpdatedLibraryConfig() {
   // Closes config stream and resets the reference to updatedConfigObserver.
   synchronized void closeConfigStream() {
     currentConfigObserver.onCompleted();
-    @Nullable
-    StreamObserver<UpdatedLibraryConfig> updatedConfigObserver = updatedConfigObserverRef.get();
-    if (updatedConfigObserver != null) {
-      updatedConfigObserver.onCompleted();
-      updatedConfigObserverRef.set(null);
-    }
+  }
+
+  private synchronized void resetUpdatedConfigObserverRef() {
```

Comment:
Be consistent with the client observer streams naming:
configRequestObserver
configResponseObserver

Reply:
Done.